api/src/db/models/user_models.py

```diff
@@ -6,43 +6,44 @@
 from sqlalchemy.orm import Mapped, mapped_column, relationship
 
 from src.adapters.db.type_decorators.postgres_type_decorators import LookupColumn
+from src.constants.lookup_constants import ExternalUserType
 from src.db.models.base import ApiSchemaTable, TimestampMixin
 from src.db.models.lookup_models import LkExternalUserType
 
 
 class User(ApiSchemaTable, TimestampMixin):
     __tablename__ = "user"
 
-    user_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True)
+    user_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True, default=uuid.uuid4)
 
 
 class LinkExternalUser(ApiSchemaTable, TimestampMixin):
     __tablename__ = "link_external_user"
 
-    link_external_user_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True)
+    link_external_user_id: Mapped[uuid.UUID] = mapped_column(primary_key=True, default=uuid.uuid4)
```

Comment:
If we need these to match to connect to an existing user do we want to make our workaround for it not existing more specific to our mock than having this just silently fill in the default? I've seen where you like MD5 the string value and then that is a valid UUID so the same identifier put through our mock twice would match the same internal user?

Reply:
This is actually something I shouldn't have added, was being a bit too copy-paste heavy with some cleanup here.

This is definitely a column that should never have a default.

Reply:
The `link_external_user_id` is just an internal ID.

The one that connects to the user in login.gov goes in the `external_user_id` field

Reply:
Happy to adjust the naming if you can help figure out a better naming scheme, I even made the same mistake a few times

Reply:
Hmm, ok, I don't have a better name... so I just missed the context of the table name. Now I'm noticing it's the PK too so that wouldn't really make sense either.